src/docfx/build/page/BuildPage.cs

```diff
@@ -281,6 +281,13 @@ private static (TemplateModel model, JObject metadata) TransformToTemplateModel(
                 pageModel.Remove("_op_gitContributorInformation");
                 pageModel.Remove("_op_allContributorsStr");
             }
+
+            // remove updated_at_date_time after applying template - legacy behavior
+            if (pageModel.TryGetValue("_op_gitContributorInformation", out var gitInfoToken) && gitInfoToken is JObject gitInfo && gitInfo.ContainsKey("updated_at_date_time"))
```

Comment:
can this be done in migration tool? It's weird to remove it here for no reason.

Reply:
yes we can, but we need know the reason why it's removed from v2 page model, @fenxu  would you please help to confirm.